### PR TITLE
Fix #386: FIDO2 Test application: Frontend

### LINKED ISSUE #386: FIDO2 Test application: Frontend

### PR DESCRIPTION
**Pull request by jnpsk**
Added two simple frontend forms to simulate login/register and payment. This PR also includes adding css files.

Login form also allows to interactively set webauthn advanced options. Those are visible after clicking on the settings button only.    
The Payment form also supports adding custom operation parameters and operation template selection for convenient operation data testing.

To simulate feel of login to an application, a simple session management was added. Every time an assertion succeed, a user details are set to the session details. In Home controller there is a redirect logic based on the session details available.

### CODE REVIEW

File: powerauth-fido2-tests/src/main/java/com/wultra/security/powerauth/fido2/controller/AssertionController.java [hunk 1]
```diff
@@ -53,8 +54,13 @@ public AssertionOptionsResponse options(@Valid @RequestBody final AssertionOptio
     }
 
     @PostMapping
-    public AssertionVerificationResponse verify(@Valid @RequestBody final VerifyAssertionRequest request) throws PowerAuthClientException {
-        return assertionService.authenticate(request);
+    public AssertionVerificationResponse verify(@Valid @RequestBody final VerifyAssertionRequest request, final HttpSession session) throws PowerAuthClientException {
+        final AssertionVerificationResponse response = assertionService.authenticate(request);
+        if (response.isAssertionValid()) {
```
Comment (banterCZ) [inline powerauth-fido2-tests/src/main/java/com/wultra/security/powerauth/fido2/controller/AssertionController.java:59]: May we log the result (success/error) or is it already done in the service?

**Review by banterCZ — APPROVED** [review]
Looks good.

File: powerauth-fido2-tests/src/main/webapp/resources/js/payment.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    if (operationTemplateList.length < 1) {
+        console.log("There is no operation template to choose from.");
+        $("#errorMessage").html("Create a payment template first.");
+        $("#errorDiv").show();
+        $(":submit").attr("disabled", true);
+    } else if (!operationTemplateList.includes("payment")) {
+        console.log("There is not operation template 'payment'.");
+    }
+
+    // Check if any application is available to select
```
[the remark this answers is not in the document]
Comment (jnpsk) [reply powerauth-fido2-tests/src/main/webapp/resources/js/payment.js]: removed

File: powerauth-fido2-tests/src/main/java/com/wultra/security/powerauth/fido2/controller/AssertionController.java (hunk 1 above)
Comment (jnpsk) [reply powerauth-fido2-tests/src/main/java/com/wultra/security/powerauth/fido2/controller/AssertionController.java:59]: The JS has verbose logging, but I added some logging to the service layer too.

File: powerauth-fido2-tests/src/main/resources/templates/login.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                        <option selected>required</option>
+                        <option>preferred</option>
+                        <option>discouraged</option>
+                    </select>
+
+                    <label for="attestation" class="label">Attestation</label>
+                    <select id="attestation" class="form-control">
+                        <option>enterprise</option>
+                        <option selected>none</option>
+                        <option>direct</option>
```
Comment (romanstrobl) [inline powerauth-fido2-tests/src/main/resources/templates/login.html:96]: Can we make the `direct` option as default? This is what we usually need for testing. Thanks.

**Review by romanstrobl — CHANGES_REQUESTED** [review]
In general nice application. However, I do not like that there are over 200 unused CSS files, can we please optimize this a bit?

**Review by romanstrobl — APPROVED** [review]
Looks OK now.

**Pull request merged**